realsense2_camera_msgs/msg/RGBD.msg

```diff
@@ -0,0 +1,6 @@
+# RGBD Message
+std_msgs/Header header
+sensor_msgs/Image rgb
```

Comment:
Don't you prefer: 
```
sensor_msgs/CameraInfo rgb_camera_info
sensor_msgs/Image rgb
sensor_msgs/CameraInfo depth_camera_info
sensor_msgs/Image depth
```

First metadata then data?

Reply:
what is the point of the order you chose ? anyway the user will fetch each field by it's id

Reply:
From the concept of first header / metadata then data.

Lets say you need to debug and view the raw data, what will you prefer? fir to see 10000000 bytes of image and then ~50 bytes of timetag / id and stuff like that?

I would even change it to 
```
sensor_msgs/CameraInfo rgb_camera_info
sensor_msgs/CameraInfo depth_camera_info
sensor_msgs/Image rgb
sensor_msgs/Image depth
```

But it's a manner of style, your call.

Reply:
Updated.